pydrad/parse/parse.py

```diff
@@ -525,6 +436,102 @@ def grid_edges_right(self) -> u.cm:
         """
         return self.grid_edges[1:]
 
+    @property
+    @u.quantity_input
+    def coordinate(self) -> u.cm:
+        """
+        Coordinate in the field-aligned direction
+        """
+        return self.grid_centers
+
+    @property
+    @u.quantity_input
+    def electron_mass_density(self) -> u.Unit('g cm-3'):
+        # TODO: Account for possible presence of both electron
+        # and ion mass densities. If the electron mass density
+        # is present in this file, it will shift all of the
+        # indices in the .amr file by one.
+        if 'electron_mass_density' in self._amr_data.colnames:
+            return self._amr_data['electron_mass_density']
+        return self.ion_mass_density
+
+    @property
+    @u.quantity_input
+    def ion_mass_density(self) -> u.Unit('g cm-3'):
+        return self._amr_data['ion_mass_density']
+
+    @property
+    @u.quantity_input
+    def momentum_density(self) -> u.Unit('g cm-2 s-1'):
+        return self._amr_data['momentum_density']
+
+    @property
+    @u.quantity_input
+    def electron_energy_density(self) -> u.Unit('erg cm-3'):
+        return self._amr_data['electron_energy_density']
+
+    @property
+    @u.quantity_input
+    def ion_energy_density(self) -> u.Unit('erg cm-3'):
+        return self._amr_data['ion_energy_density']
+
+    @property
+    @u.quantity_input
+    def velocity(self) -> u.cm/u.s:
+        if hasattr(self, '_phy_data'):
+            return self._phy_data['velocity']
+        return self.momentum_density / self.ion_mass_density
+
+    @property
+    @u.quantity_input
+    def ion_density(self) -> u.Unit('cm-3'):
+        if hasattr(self, '_phy_data'):
+            return self._phy_data['ion_density']
+        # NOTE: This is pulled directly from the HYDRAD source code
+        # https://github.com/rice-solar-physics/HYDRAD/blob/master/Resources/source/constants.h
+        # and is the average ion mass for a H-He plasma computed by weighting the H and He ion
+        # masses with a particular set of abundances.
+        m_ion = 2.171e-24 * u.g
+        return self.ion_mass_density / m_ion
+
+    @property
+    @u.quantity_input
+    def electron_density(self) -> u.Unit('cm-3'):
+        if hasattr(self, '_phy_data'):
+            return self._phy_data['electron_density']
+        # FIXME: If this exists as a separate column in the .amr file then
+        return self.ion_density
+
+    @property
+    @u.quantity_input
+    def electron_pressure(self) -> u.Unit('dyne cm-2'):
+        if hasattr(self, '_phy_data'):
+            return self._phy_data['electron_pressure']
+        gamma = 5/3
```

Comment:
Maybe this shouldn't be hard coded?  I think it's also found in the constants.h file.  I don't think it would ever be updated, so probably not worrisome.

Reply:
Agreed. I was mildly annoyed that this isn't already defined in `astropy.constants`. We could create a constants file and stick it in there.

Reply:
I'd say astropy should add it, but the truth is that this isn't a constant, so much as a value specific to monatomic gas.  

A constants file is probably a reasonable way to deal with this.